cmd2.py

```diff
@@ -549,17 +538,18 @@ def cmd_wrapper(instance, cmdline):
         # we want it to be the name of our command
         argparser.prog = func.__name__[3:]
 
-        # put the help message in the method docstring
-        if func.__doc__:
+        # If the description has not been set, then use the method docstring if one exists
```

Comment:
So an argparse description will now override the docstring?  That sounds OK with me.  You might want to update the Sphinx documentation and/or examples to show this behavior.